src/mix_paula.c

```diff
@@ -125,21 +125,25 @@ static void do_clock(struct paula_state *paula, int cycles)
     unsigned int pos = vi->pos; \
     int frac = (1 << SMIX_SHIFT) * (vi->pos - (int)vi->pos)
 
+#define VAR_PAULA_MONO(x) \
+    VAR_NORM(x); \
+    vl <<= 8
+
```

Comment:
Don't touch it, and no warnings  -  huh..

Reply:
Right, because `-Wno-unused-parameter` is used. For some reason the second you touch it, it triggers a *completely different* warning...